src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesDeclarativeAgent.java

```diff
@@ -139,10 +145,10 @@ public String getPodTemplateName() {
     @DataBoundSetter
     public void setPodTemplateName(String podTemplateName) {
         this.podTemplateName = podTemplateName;
-        if (!StringUtils.isEmpty(podTemplateName)) {
+        if (podTemplateName != null) {
```

Comment:
missing `this.podTemplateName = Utils.fixEmpty(podTemplateName)` ?

Reply:
not sure where `Utils.fixEmpty` comes from or why its used? Is it to use an empty string if the value is null?

Reply:
`Util` sorry, yes it's for that so you store null to avoid extra checks later on

http://javadoc.jenkins.io/hudson/Util.html#fixEmpty-java.lang.String-

Reply:
thx - fixed!